spec/features/create_extension_spec.rb

```diff
@@ -13,7 +13,8 @@
   let(:gemspec_name) { "solidus_#{extension_name}.gemspec" }
   let(:tmp_path) { Pathname.new(ext_root).join('spec', 'tmp') }
   let(:install_path) { tmp_path.join("solidus_#{extension_name}") }
-  let(:command_failed) { Class.new(StandardError) }
+
+  class CommandFailed < StandardError; end
```

Comment:
@aldesantis qualche motivo specifico per questa modifica? 
Facendo così CommandFailed viene definito su Object.

_Essendo nelle spec non è un grosso problema, ma come era prima pure aveva senso._

Reply:
@elia leads to better error messages, but also happy to revert to the previous one. I'm not too afraid of polluting the namespace here tbh, but let me know.

Reply:
@aldesantis we could pull the error class along with most helpers to a support/ helper module, including that in the spec we should both solve the problem and cleanup the file.

Reply:
I created an issue, so we can mark this as solved ✔️